lv2/CMakeLists.txt

```diff
@@ -11,7 +11,13 @@ set (LV2PLUGIN_TTL_SRC_FILES
     manifest.ttl.in
     ${PROJECT_NAME}.ttl.in
 )
-add_library (${LV2PLUGIN_PRJ_NAME} MODULE ${PROJECT_NAME}.c ${LV2PLUGIN_TTL_SRC_FILES})
+source_group("Turtle Files" FILES
```

Comment:
This creates groups in the Qt project?

Reply:
Yes, by default QtCreator creates a "Source Files" and a "Header Files" groups by default based on the files listed in the CMakeLists.txt, that one would be a custom group to keep other file types separated.